sys/auto_init/auto_init.c

```diff
@@ -7,7 +7,7 @@
  * General Public License v2.1. See the file LICENSE in the top level
  * directory for more details.
  *
- * @ingroup auto_init
+ * @ingroup sys_autoinit
```

Comment:
Here and below: shouldn't it rather be `sys_auto_init`?

Reply:
I kept the initially defined `sys_autoinit`, I'll use your suggestion.

Reply:
name changed